### PR TITLE
Added Core SE OEM version [REVPI-3018]

### PR DESCRIPTION
**Pull request by hjokunbus**
The device description has to be added in order to use HAT EEPROM of OEM version of Core SE properly.

### CODE REVIEW

**Comment by iluminat23** [comment]
Should we add the Vendor to the README.md? Now we have different RevPi Core SE 8GB in the list.

**Comment by iluminat23** [comment]
And since we don't use symlinks anymore we have a hard time to actually see which configs are the same.

File: README.md [hunk 1]
```diff
@@ -14,3 +14,4 @@ Device specific JSON templates for RevPi HAT EEPROM generation
 | [RevPi Core SE 8GB](./revpi-hat-PR100365R00.json) | PR100365 | R00 | 1 | revpi-core-se-2022 |
 | [RevPi Core SE 16GB](./revpi-hat-PR100366R00.json) | PR100366 | R00 | 1 | revpi-core-se-2022 |
 | [RevPi Core SE 32GB](./revpi-hat-PR100367R00.json) | PR100367 | R00 | 1 | revpi-core-se-2022 |
+| [RevPi Core SE 8GB](./revpi-hat-PR100375R00.json) | PR100365 | R00 | 1 | revpi-core-se-2022 |
```
Comment (nbuchwitz) [inline README.md:17]:
```suggestion
| [RevPi Core SE 8GB (OEM DV8)](./revpi-hat-PR100375R00.json) | PR100365 | R00 | 1 | revpi-core-se-2022 |
```

File: revpi-hat-PR100375R00.json [hunk 2]
```diff
@@ -0,0 +1,174 @@
+{
+    "version": 1,
+    "eeprom_data_version": 1,
+    "vstr": "DV8 PRODUCTS INC.",
+    "pstr": "RevPi Core SE 8GB",
```
Comment (nbuchwitz) [inline revpi-hat-PR100375R00.json:5]:
```suggestion
    "pstr": "RevPi Core SE 8GB (OEM DV8)",
```

File: revpi-hat-PR100375R00.json [hunk 3]
```diff
@@ -0,0 +1,174 @@
+{
+    "version": 1,
+    "eeprom_data_version": 1,
+    "vstr": "DV8 PRODUCTS INC.",
```
Comment (nbuchwitz) [inline revpi-hat-PR100375R00.json:4]:
```suggestion
    "vstr": "DV8 Energy",
```

**Review by nbuchwitz — CHANGES_REQUESTED** [review]

**Review by nbuchwitz — APPROVED** [review]

**Pull request merged**